frontend/pages/index.tsx

```diff
@@ -35,7 +35,7 @@ export default function Home({ sponsors }: Sponsors) {
       <main className={montserrat.className}>
         <Navbar />
         <Hero />
-        <Countdown />
+        <Countdown date={new Date()}  />
```

Comment:
Remove date param, as it will always ask for current date. Better to be handled inside of the countdown file.